tests/test_api.py

```diff
@@ -315,3 +315,33 @@ async def content(request, page):
     assert response_one.text == "one"
     assert response_two.text == "two"
     assert respx.stats.call_count == 2
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    "method_str, client_method_attr",
+    [
+        ("DELETE", "delete"),
+        ("delete", "delete"),
+        ("GET", "get"),
+        ("get", "get"),
+        ("HEAD", "head"),
+        ("head", "head"),
+        ("OPTIONS", "options"),
+        ("options", "options"),
+        ("PATCH", "patch"),
+        ("patch", "patch"),
+        ("POST", "post"),
+        ("post", "post"),
+        ("PUT", "put"),
+        ("put", "put"),
+    ],
+)
+async def test_can_add_by_lowercase_method(client, method_str, client_method_attr):
```

Comment:
I saw that we actually don't test `add` by it self with http method stings.

This test should be that, suggesting renaming the test case more generic.

```suggestion
async def test_add(client, method_str, client_method_attr):
```